src/FixedDataTable.js

```diff
@@ -920,7 +920,7 @@ var FixedDataTable = React.createClass({
     }
 
     var lastScrollTop = oldState ? oldState.scrollTop : undefined;
-    if (props.scrollTop !== lastScrollTop) {
+    if (typeof props.scrollTop !== 'undefined' && props.scrollTop !== null && props.scrollTop !== lastScrollTop) {
```

Comment:
It's a habit I suppose, but it's usually considered safer.  In strict mode accessing a variable that is undefined will throw a referenceError.  That may not apply in this case, but as a general rule to be safe that's how I like to check for undefined variables.